doc/users/cheatsheets/matplotlib_for_matlab_users.rst

```diff
@@ -0,0 +1,71 @@
+.. _matplotlib-for-matlab-users:
+
+================================
+WIP: Matplotlib for Matlab users
+================================
+
+Introduction
+============
+
+If it is not obvious enough from its name, the Matplotlib was originally
+developped as a migration and improvement of the namesake MATLAB® plotting
+utilies in Python. Throughout its :doc:`/users/history`, lots
+of effort was made to  strike a resembling interface to MATLAB®. After more
+than a decade of development, Matplotlib has become a major part of the Python
+ecosystem. It grows far beyond its original purpose of a MATLAB®
+plotting emulater. This page is intended to be a reference for new matplotlib
+users who are coming from MATLAB®. It collects common mistakes and useful
+wisdoms to help the transition.
+
+.. raw:: html
+
+   <style>
+   table.docutils td { border: solid 1px #ccc; }
+   </style>
+
+First Things First
+====================
+WIP:
+
+Insert two code blocks: one being the "hello world" plot from matplotlib and
+the other one from MATLAB®. It'll be nice to show code and figures side
+by side in a two-column table. But I'm not sure how easy it is to acheive it
+using RST.
+
```

Comment:
That's a great idea/ totally support this. You may be able to do this with tables in sphinx https://www.sphinx-doc.org/en/1.8/usage/restructuredtext/directives.html#tables